addons/hr_holidays/report/holidays_summary_report.py

```diff
@@ -47,7 +47,7 @@ def _get_months(self, start_date):
             start_date += relativedelta(day=1, months=+1)
         return res
 
-    def _get_leaves_summary(self, start_date, empid, holiday_type):
+    def _get_leaves_summary(self, employee, start_date, holiday_type):
```

Comment:
```suggestion
    def _get_leaves_summary(self, start_date, employee, holiday_type):
```
This avoid to change parameter order and do not break inheritance

Reply:
You have got a point here.
I may keep the input parameters as before, but the output result *must change*.
So all the method overrides that *use* the output must be adapted.